README.md

```diff
@@ -8,6 +8,16 @@
 
 ## Installation
 
+Add amphp/beanstalk to your composer.json file or you can require amphp/beanstalk with composer. If you are not using Composer, you should be. It's an excellent way to manage dependencies in your PHP application.
+
 ```bash
 composer require amphp/beanstalk
 ```
+
+Then from the command line:
+```bash
+composer install
+```
```

Comment:
`composer require` automatically installed new dependencies. Maybe just change it to the version of https://github.com/amphp/amp#installation with a link to Composer?

Reply:
you mean replace this scope installation with url  https://github.com/amphp/amp#installation ?

Reply:
Well, the same text, but with this package of course. :+1:

Reply:
ah i see, oke i can make new pr :)